app/Models/Trip.php

```diff
@@ -66,11 +71,11 @@ public function polyline(): HasOne {
     }
 
     public function originStation(): BelongsTo {
-        return $this->belongsTo(Station::class, 'origin', 'ibnr');
+        return $this->belongsTo(Station::class, 'origin_id', 'id');
```

Comment:
Geht das in jedem Fall? Das setzt ja voraus, dass jeder Trip schon aktualisiert wurde, dass uns die relations nicht kaputt gehen, oder sehe ich das falsch?

Reply:
Hab weiter unten die Migration gesehen. Nvm. :D